src/averageAges.js

```diff
@@ -14,12 +14,27 @@
  *
  * @return {number}
  */
+
+function getAverageAge(humans) {
+  const agesMap = humans.map(person => person.died - person.born);
+  const averageAges = agesMap.reduce(
+    (prev, next) => prev + next,
+    0) / agesMap.length;
+
+  return averageAges;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const toFilterMan = (person) => {
+    const isMan = person.sex === 'm';
+    const isCentury = Math.ceil(person.died / 100) === century;
+
+    return century ? isMan && isCentury : isMan;
+  };
+
+  const filteredMan = people.filter(person => toFilterMan(person));
```

Comment:
```suggestion
  const filteredMan = people.filter(toFilterMan);
```

Reply:
I've explained it on todays lesson, just pass function to filter method, it will take needed arguments, why to create function wrapper?